agent/main.go

```diff
@@ -86,7 +86,7 @@ func main() {
 	flag.StringVar(&namespace, "namespace", "default", "Namespace in the management cluster where you would like to register this host")
 	flag.Var(&labels, "label", "labels to attach to the ByoHost CR in the form labelname=labelVal for e.g. '--label site=apac --label cores=2'")
 	flag.StringVar(&metricsbindaddress, "metricsbindaddress", ":8080", "metricsbindaddress is the TCP address that the controller should bind to for serving prometheus metrics.It can be set to \"0\" to disable the metrics serving")
-	flag.StringVar(&downloadpath, "downloadpath", "/var/lib", "File System path to keep the downloads")
+	flag.StringVar(&downloadpath, "downloadpath", "/var/lib/kubernetes-artifacts", "File System path to keep the downloads")
```

Comment:
Question: In my understanding, this path is not a preexisting one. Is the creation of the default path if not existing is already handled in installer code?

Reply:
Yup
https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/5900e9d1b2b744334689c5a9ad4b8c13c2cd2826/agent/installer/bundle_downloader.go#L144-L147

Reply:
You need to put the product/project name right after /var/lib and then the rest.

Reply:
Technically speaking **bundles** vs kubernetes-artifacts will be better because:
- in our doc we refer to bundles
- a bundle has containerd and is "more" than kubernetes

Reply:
Done. PTAL.